infrastructures/infrastructure-azure/src/test/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructureTest.java

```diff
@@ -0,0 +1,353 @@
+                   is("http://" + azureInfrastructure.rmHostname + ":8080/connector-iaas"));
+        assertThat(azureInfrastructure.image, is(nullValue()));
+        assertThat(azureInfrastructure.vmSizeType, is(nullValue()));
+        assertThat(azureInfrastructure.vmUsername, is(nullValue()));
+        assertThat(azureInfrastructure.vmPassword, is(nullValue()));
+        assertThat(azureInfrastructure.vmPublicKey, is(nullValue()));
+        assertThat(azureInfrastructure.resourceGroup, is(nullValue()));
+        assertThat(azureInfrastructure.region, is(nullValue()));
+        assertThat(azureInfrastructure.numberOfInstances, is(1));
+        assertThat(azureInfrastructure.numberOfNodesPerInstance, is(1));
+        if (System.getProperty("os.name").contains("Windows")) {
+            assertThat(azureInfrastructure.downloadCommand,
+                       is("powershell -command \"& { (New-Object Net.WebClient).DownloadFile('http://" +
+                          azureInfrastructure.rmHostname + ":8080/rest/node.jar', 'node.jar') }\""));
+        } else {
+            assertThat(azureInfrastructure.downloadCommand,
+                       is("wget -nv http://" + azureInfrastructure.rmHostname + ":8080/rest/node.jar"));
+
+        }
+        assertThat(azureInfrastructure.privateNetworkCIDR, is(nullValue()));
+        assertThat(azureInfrastructure.staticPublicIP, is(true));
+        assertThat(azureInfrastructure.additionalProperties, is("-Dproactive.useIPaddress=true"));
+
+    }
+
+    @Test
+    public void testConfigure() {
+        when(nodeSource.getName()).thenReturn("Node source Name");
+        azureInfrastructure.nodeSource = nodeSource;
+
+        azureInfrastructure.configure("clientId",
+                                      "secret",
+                                      "domain",
+                                      "subscriptionId",
+                                      "authenticationEndpoint",
+                                      "managementEndpoint",
+                                      "resourceManagerEndpoint",
+                                      "graphEndpoint",
+                                      "test.activeeon.com",
+                                      "http://localhost:8088/connector-iaas",
+                                      "image",
+                                      "Standard_D1_v2",
+                                      "vmUsername",
+                                      "vmPassword",
+                                      "vmPublicKey",
+                                      "resourceGroup",
+                                      "region",
+                                      "2",
+                                      "3",
+                                      "wget -nv test.activeeon.com/rest/node.jar",
+                                      "192.168.1.0/24",
+                                      true,
+                                      "-Dnew=value");
```

Comment:
Yes this code should throw an exception in case of a missing parameter so there is nothing to test, I still added a try/catch block with explicit succeed/failure asserts in 06b250351a3cac447f5c0b0272c314c5bd90a0cb